paywall/src/utils/validators.js

```diff
@@ -212,6 +212,12 @@ export const isValidPaywallConfig = config => {
     return false
   }
 
+  if (config.metadataInputs) {
+    if (!isValidMetadataArray(config.metadataInputs)) {
```

Comment:
could we return `isValidMetadataArray(config.metadataInputs)` here ?

Reply:
We can't, because we can't return `true` unless every condition passes. If we hypothetically added other validations later that come after this one, that could be a subtle bug because the later additions would never get checked.